.golangci.yml

```diff
@@ -110,6 +110,7 @@ linters-settings:
           # Approved packages.
           - "github.com/spf13/cobra" # For CLI
           - "github.com/coreos/go-oidc" # For verifying OIDC tokens.
+          - "github.com/go-jose/go-jose/v4" # For testing OIDC tokens
```

Comment:
hmm. How was depguard allowing `gopkg.in/square/go-jose.v2` before? 🤔

Reply:
my guess is it was an old line, never linted? but I really don't know